src/components/character-count/index.md.njk

```diff
@@ -107,7 +112,9 @@ During this research, the component was tested with 17 users, including those wi
 
 ### Known issues and gaps
 
-There is a known issue with JAWS where the field’s label is repeated every time the count message is played. This is repetitive and likely to be disruptive for users.
+In Internet Explorer 11, JAWS will ignore any set threshold and announce the character count, even if the user entered less than the threshold.
+
+In Chrome version 99, JAWS will not announce the hint or character count of a pre-populated textarea. This is a known issue in JAWS.
```

Comment:
Can we emphasise by saying it's "widely known", or even give a link? Otherwise seems like a redundant under a heading that already says 'Known issues'

```suggestion
In Chrome version 99, JAWS will not announce the hint or character count of a pre-populated textarea. This is a widely known issue in JAWS.
```

Reply:
Hey @calvin-lau-sig7 we can give a link, and I think it would be great to have a link. I just wasn't sure if that's a standard thing we do in guidance content.

The link would be https://github.com/FreedomScientific/VFO-standards-support/issues/201

Reply:
Yeah, that's fine! I think we've linked to Apple and Google's bug trackers before. I'll add a bit of context to the link as I never knew who made JAWS and suspect that might be useful.